src/index.html

```diff
@@ -5,9 +5,62 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+      >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+      class="card__img"
+      src="./images/imac.jpeg"
+      alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+      width="160"
+      height="134"
+      >
```

Comment:
```suggestion
      <img
        class="card__img"
        src="./images/imac.jpeg"
        alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
      >
```
Don't use inline styles, specify properties in the css/scss files with class.
[CODE STYLE] - If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.